.github/docs/README.md

```diff
@@ -206,13 +221,16 @@ import cats.effect.Resource
 
 import org.http4s.client.Client
 import org.http4s.Response
+import org.http4s.Uri
 
 val httpClient: Client[IO] = Client[IO] { _ => Resource.pure(Response[IO]())}
 val config = Config(TokenType.UserAccount)
+val myAudience = Uri.unsafeFromString("https://my-run-app.a.run.app")
```

Comment:
q: why is that called myAudience?

Reply:
so it doesn't collide with an existing `audience` 😂 , still wanted to keep it separated in case it changes in the future